js/main.js

```diff
@@ -4,9 +4,37 @@
 var $photoimg = document.querySelector('#entry-img');
 var $photoURL = document.querySelector('#photo');
 var $form = document.querySelector('#input-form');
+var $entriesPage = document.querySelector('#entries-page');
+var $entriesFormPage = document.querySelector('#entry-form');
+var $noEntriesText = document.querySelector('#no-entries-text');
+var $list = document.querySelector('ul');
 
 $photoURL.addEventListener('input', handleInput);
 $form.addEventListener('submit', handleSubmit);
+window.addEventListener('DOMContentLoaded', contentLoaded);
+window.addEventListener('click', switchView);
+
+function contentLoaded(event) {
+  if (data.entries.length !== 0) {
+    $noEntriesText.className = 'hidden page column-full flex jc-center';
+  } else {
+    $noEntriesText.className = 'page column-full flex jc-center';
+  }
+  for (var i = 0; i < data.entries.length; i++) {
+    $list.appendChild(renderEntry(data.entries[i]));
+  }
+}
+
+function switchView(view) {
+  var $dataView = event.target.getAttribute('data-view');
+  if ($dataView === 'entries') {
+    $entriesPage.className = 'page';
+    $entriesFormPage.className = 'page hidden';
+  } else if ($dataView === 'entry-form') {
+    $entriesPage.className = 'page hidden';
+    $entriesFormPage.className = 'page';
+  }
```

Comment:
Instead of having an individual block for each possible view, you should do this with a loop. Loop over all view containers, and find the one that has the same name as the value stored in the `view` parameter. If they match, show that view, and if they don't match, hide that view.